Project.toml

```diff
@@ -62,7 +62,7 @@ LaTeXStrings = "1.3.0"
 Latexify = "0.16.6"
 MacroTools = "0.5.5"
 Makie = "0.22.1"
-ModelingToolkit = "9.73"
+ModelingToolkit = "9.73, 10"
```

Comment:
We should drop MTK < 10. I don't think Catalyst will work with both after these changes.